### ISSUE TITLE
Update the readme file with details about what is NLP.

### ISSUE THREAD

**Issue by animeshsrivastava24** [animeshsrivastava24/New-Dictionary-Words#2]

**Comment by mohancm** [comment]
is that about Neuro-linguistic programming? @animeshsrivastava24

**Comment by codedoper45** [comment]
I guess @animeshsrivastava24  requires just information about NLP in the readme?
Am I right?

**Comment by animeshsrivastava24** [comment]
Yes, @codedoper45  just submit information about NLP, @mohancm  Yeah you are open to submit information of any subfield of NLP (Natural Lang. Processing).
But Neuro-linguistic programming sounds new to me, can you please write details about it the Readme
